external/CMakeLists.txt

```diff
@@ -234,6 +236,7 @@ ExternalProject_Add(
     -DRDK_INSTALL_STATIC_LIBS=${RDKIT_INSTALL_STATIC_LIBS_VAL}
     -DRDK_OPTIMIZE_POPCNT=OFF
     -DRDK_USE_BOOST_STACKTRACE=OFF
+    -DRDK_BUILD_CHEMDRAW_SUPPORT=OFF
```

Comment:
Do we need to block .cdxml read support since this is now turned off?

Reply:
Yes.